src/query/settings/src/settings_default.rs

```diff
@@ -624,6 +631,24 @@ impl DefaultSettings {
                     mode: SettingMode::Both,
                     range: Some(SettingRange::Numeric(0..=1)),
                 }),
+                ("cost_factor_hash_table_per_row", DefaultSettingValue {
+                    value: UserSettingValue::UInt64(COST_FACTOR_HASH_TABLE_PER_ROW),
+                    desc: "Cost factor of building hash table for a data row",
+                    mode: SettingMode::Both,
+                    range: Some(SettingRange::Numeric(0..=u64::MAX)),
+                }),
+                ("cost_factor_aggregate_per_row", DefaultSettingValue {
```

Comment:
These new settings is for developer or the user? The `desc` is not easy to understand how to use them.

Reply:
Yes, it’s for developers. Do we have solution not to expose these settings to users?

Reply:
Yes, we don't recommand export the `VALUE` setting, if they set the global value it hard to adjust it in the future. We should atomic setting them by us not setting.

Reply:
> Settings is good for: disable or enable a feature by us.

I need a way to modify the configuration dynamicly to tune the cost model. Do you have any suggestion for this?

Reply:
Ok, I see. This PR works.

Reply:
BTW, we can not tell the user to set these settings, only used by us.